config/registry_image_pruner/cronjob.yaml

```diff
@@ -33,7 +33,18 @@ spec:
             volumeMounts:
               - name: image-pruner-volume
                 mountPath: /image-pruner
+            resources:
+              limits:
+                cpu: 500m
+                memory: 512Mi
+              requests:
+                cpu: 150m
+                memory: 128Mi
+            securityContext:
+              readOnlyRootFilesystem: true
```

Comment:
@mmorhun Looks like I did not test properly the cronjob. It is failing in staging because the container cannot write to the rootfilesystem:
```
ERROR: Exception:
Traceback (most recent call last):
  File "/opt/app-root/lib64/python3.9/site-packages/pip/_internal/cli/base_command.py", line 167, in exc_logging_wrapper
    status = run_func(*args)
  File "/opt/app-root/lib64/python3.9/site-packages/pip/_internal/cli/req_command.py", line 247, in wrapper
    return func(self, options, args)
  File "/opt/app-root/lib64/python3.9/site-packages/pip/_internal/commands/install.py", line 324, in run
    wheel_cache = WheelCache(options.cache_dir, options.format_control)
  File "/opt/app-root/lib64/python3.9/site-packages/pip/_internal/cache.py", line 227, in __init__
    self._ephem_cache = EphemWheelCache(format_control)
  File "/opt/app-root/lib64/python3.9/site-packages/pip/_internal/cache.py", line 195, in __init__
    self._temp_dir = TempDirectory(
  File "/opt/app-root/lib64/python3.9/site-packages/pip/_internal/utils/temp_dir.py", line 125, in __init__
    path = self._create(kind)
  File "/opt/app-root/lib64/python3.9/site-packages/pip/_internal/utils/temp_dir.py", line 164, in _create 
    path = os.path.realpath(tempfile.mkdtemp(prefix=f"pip-{kind}-"))
  File "/usr/lib64/python3.9/tempfile.py", line 352, in mkdtemp
    prefix, suffix, dir, output_type = _sanitize_params(prefix, suffix, dir)
  File "/usr/lib64/python3.9/tempfile.py", line 122, in _sanitize_params
    dir = gettempdir()
  File "/usr/lib64/python3.9/tempfile.py", line 291, in gettempdir
    tempdir = _get_default_tempdir()
  File "/usr/lib64/python3.9/tempfile.py", line 223, in _get_default_tempdir
    raise FileNotFoundError(_errno.ENOENT,
FileNotFoundError: [Errno 2] No usable temporary directory found in ['/tmp', '/var/tmp', '/usr/tmp', '/opt/app-root/src']
```
I will revert that settings for now but the image-pruner will need to be changed to not write to rootFS.

Reply:
https://github.com/redhat-appstudio/image-controller/pull/65